ChatApp/View/ViewControllerConstants.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  ViewControllerConstants.swift
+//  ChatApp
+//
+//  Created by bakhva  on 28.04.23.
+//
+
+import UIKit
+
+extension ViewController {
```

Comment:
can be better if constants will be in the same file and make it private extension